Sources/Valet/MigrationError.swift

```diff
@@ -20,16 +20,11 @@
 
 import Foundation
 
+
```

Comment:
nit: Empty extra space?

Reply:
Welcome to Square's style guide. Two empty newlines between imports and code 😉